qemu/tests/balloon_check.py

```diff
@@ -24,12 +24,14 @@ def __init__(self, test, params, env):
 
         self.vm = env.get_vm(params["main_vm"])
         self.session = self.get_session(self.vm)
+        self.env["memory_stable"] = False
```

Comment:
Why put `memory_stable` into env?
If this param will be used for all balloon cases, may be we should init in 'env'